### PR TITLE
Add utility function `to_json_dict`

### PR DESCRIPTION
**Pull request by ollityrkko**
The function can be used to sanitise data (e.g. circuit metadata) before sending it IQMClient.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+Version 11.7
+============
+
+* Add utility function ``to_json_dict`` to convert a dict to a JSON dict. `#76 <https://github.com/iqm-finland/iqm-client/pull/76>`_
```
Comment (kukushechkin) [inline CHANGELOG.rst:8]: PR #77

File: src/iqm_client/util.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+class IQMJSONEncoder(JSONEncoder):
+    """JSONEncoder that that adds support for some non-JSON datatypes"""
+
+    def default(self, o: Any):
+        if isinstance(o, np.ndarray):
+            return o.tolist()
+        return JSONEncoder.default(self, o)
+
+
+def to_json_dict(obj: dict[str, Any]) -> dict:
```
Comment (kukushechkin) [inline src/iqm_client/util.py:32]: Is it being used anywhere in `iqm-client`?

File: CHANGELOG.rst (hunk 1 above)
Comment (ollityrkko) [reply CHANGELOG.rst]: fixed

File: src/iqm_client/util.py (hunk 2 above)
Comment (ollityrkko) [reply src/iqm_client/util.py:32]: It is not used anywhere in iqm-client, and it is not really intended for users as such, but rather for integrators.
First use will be in qiskit-on-iqm.
I added a short paragraph of it in integration guide.

**Pull request merged**